api/src/migrations/main/1721691825980-project-plan-table.js

```diff
@@ -0,0 +1,79 @@
+const { MigrationInterface, QueryRunner } = require("typeorm");
+
+module.exports = class ProjectPlanTable1721691825980 {
+
+    async up(queryRunner) {
+        console.log('Starting new project_plan_code migration.');
+        await queryRunner.query(`
+        -- add new table - project_plan_code
+            drop table if exists app_fom.project_plan_code;  
+            create table if not exists app_fom.project_plan_code  
```

Comment:
Hi @basilv do you feel the table "**project_plan_code**" fit the business context or do you feel strange and have a better suggestion? I don't mind to refactor it.

Reply:
I'm fine with the name.